client/src/routes/+page.svelte

```diff
@@ -24,18 +24,8 @@
 			<Form network={parachain ?? -1} />
 		</Card>
 	</div>
-
-	<p class="read-the-docs">
-		Check out <a href="https://use.ink/" target="_blank" rel="noreferrer">Ink!</a>, the official
-		Substrate documentation!
-	</p>
 </main>
 
-<style>
```

Comment:
please, undo this line. The closing style labels is missing the opening one.

Reply:
Ah - my bad 🤒 ! Fixed.